readthedocs/templates/projects/integration_webhook_detail.html

```diff
@@ -46,45 +42,47 @@
           You can use the following address to manually configure this webhook.
         {% endblocktrans %}
       </p>
-    {% else %}
+    {% endif %}
+  {% endif %}
+
+  <p>
+    {% blocktrans trimmed %}
+      To manually configure this webhook with your provider, use the
+      following information:
+    {% endblocktrans %}
+  </p>
+
+  <p>
+    {% url 'api_webhook' project_slug=project.slug integration_pk=integration.pk as webhook_url %}
+    <a href="//{{ PRODUCTION_DOMAIN }}{{ webhook_url }}">{{ PRODUCTION_DOMAIN }}{{ webhook_url }}</a>
+  </p>
+
+  {% block integration_details %}
+    <div class="integration-details">
+      {% if integration.secret %}
       <p>
+        <label for="id_secret">Secret:</label>
```

Comment:
Not a big deal with this dashboard eventually going away, but this would normally need translation:

```suggestion
        <label for="id_secret">{% trans "Secret" %}:</label>
```